cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/task/CopyNativeLibTask.java

```diff
@@ -138,7 +144,35 @@ private void copyImportedLib(BuildContext buildContext, BPackageSymbol importz,
             }
         }
 
-        // If balo cannot be found from target or cache, get dependencies from distribution toml.
+        // If platform libs are defined, copy them to target
+        List<Library> libraries = manifest.getPlatform().libraries;
+        if (libraries != null && libraries.size() > 0) {
+            List<Path> libs = libraries.stream()
```

Comment:
shall we use the same for loop? instead of streams?

Reply:
ack, done